src/js/popup.js

```diff
@@ -152,6 +152,18 @@ function init() {
     e.preventDefault();
   });
 
+  var shareOverlay = $("#share_overlay");
+
+  $("#share").on("click", share);
+  $("#share_close").on("click", function() {
+    shareOverlay.toggleClass('active', false);
+  });
+  $("#copy_button").on("click", function() {
+    $("#share_output").select();
+    document.execCommand('copy');
```

Comment:
Does this work in Firefox on Android too?

Reply:
According to https://developer.mozilla.org, while this usage is depricated, it does work on our 3 target browsers, including older versions than the [Clipboard API]( https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API).  I could change it if you think the API is a better choice.

Reply:
No, it's fine as is, thank you. As long as it works at this time.